package-lock.json

```diff
@@ -2106,7 +2106,7 @@
     },
     "@types/valid-url": {
       "version": "1.0.2",
-      "resolved": "https://registry.npmjs.org/@types/valid-url/-/valid-url-1.0.2.tgz",
+      "resolved": "http://registry.npmjs.org/@types/valid-url/-/valid-url-1.0.2.tgz",
```

Comment:
not needed changes

Reply:
Done.

Reply:
This is still showing some permission-mod related changes like "100644 → 100755". Can't we get rid of that as well.